src/commands/_common.ts

```diff
@@ -4,8 +4,12 @@ import { parse } from "pg-connection-string";
 
 import { Settings } from "../settings";
 
-export const GMRC_PATH = `${process.cwd()}/.gmrc`;
-export const GMRCJS_PATH = `${GMRC_PATH}.js`;
+export const makeGmrcJsPath = (gmrcPath: string): string => `${gmrcPath}.js`;
+export const DEFAULT_GMRC_PATH = `${process.cwd()}/.gmrc`;
+export const DEFAULT_GMRCJS_PATH = makeGmrcJsPath(DEFAULT_GMRC_PATH);
+
+// Used to type `argv` in all commands
+export type ConfigOptions = { config: string };
```

Comment:
I think this should be optional?
```suggestion
export type ConfigOptions = { config?: string };
```